src/main/java/org/jitsi/jicofo/Participant.java

```diff
@@ -51,9 +54,10 @@
      * @param chatRoomMember XMPP MUC chat room member which represents a
      *                       <tt>Participant</tt>.
      */
-    public static String getEndpointId(ChatRoomMember chatRoomMember)
+    public static Jid getEndpointId(XmppChatMember chatRoomMember)
     {
-        return chatRoomMember.getName(); // XMPP MUC Nickname
+        //return chatRoomMember.getName(); // XMPP MUC Nickname
+        return chatRoomMember.getJabberID();
```

Comment:
This looks wrong. getJabberID lacks proper documentation but looking at some logs it seems to contain a full JID. For endpoint IDs we probably want just the resource part (of the MUC JID), which is also what getName() seems to return.

Reply:
My comment above is obsolete, I didn't see all the commits. Please disregard.